dev/io.openliberty.com.ibm.ws.jsf/src/com/ibm/ws/jsf/cdi/package-info.java

```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
```

Comment:
You updated the other copyright dates since we moved this existing class to a new bundle but didn't update it here.